test/test.js

```diff
@@ -793,7 +793,7 @@ describe('S3rver Tests with Static Web Hosting', function () {
         return done(new Error('Invalid status: ' + response.statusCode));
       }
 
-      if (response.headers['content-type'] !== 'text/html; charset=utf-8') {
+      if (response.headers['content-type'] !== 'text/html') {
```

Comment:
The previous content-type was wrong.
I created a bucket myself and enabled static web hosting. This is the correct content-type you should receive.